test/e2e/federated-service.go

```diff
@@ -34,6 +35,9 @@ const (
 
 	FederatedServiceName    = "federated-service"
 	FederatedServicePodName = "federated-service-test-pod"
+
+	KubeDNSConfigMapName      = "kube-dns"
```

Comment:
Nit. You can fix this in a follow up. Since all our e2e tests except the node ones are in the same package (`e2e`), it is better to prefix them with the test or test group name to avoid namespace pollution.